integration-tests/http/__tests__/shipping-option/admin/shipping-option.spec.ts

```diff
@@ -359,6 +501,22 @@ medusaIntegrationTestRunner({
             rules: [shippingOptionRule],
           }
 
+          const remoteLink = appContainer.resolve(
+            ContainerRegistrationKeys.REMOTE_LINK
+          )
+
+          // TODO: move this to an endpoint when available
+          await remoteLink.create([
```

Comment:
suggestion: Maybe create a helper func so we have one place to update when the endpoint is available

Reply:
I've already replaced these with the endpoint in the new branch i'm working on 🤞🏻